dx/usr/bin/bluefin-dx-user-setup

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Including this check here as well as a failsafe
+if [[ -f $USER_SETUP_VER_FILE && "$USER_SETUP_VER" = "$USER_SETUP_VER_RAN" ]]; then
+  echo "User setup has already run. Exiting..."
+  exit 0
+fi
+
+# Setup VSCode
+# Pre-install preferred VSCode Extensions
+code --install-extension ms-vscode-remote.remote-containers
```

Comment:
Since this would need internet access, I'd suggest creating a separate user service that runs after network-online.target otherwise the service will fail if someone boots for the first time and doesn't have an active connection

Something like this:

```
[Unit]
Description=Sets up VS Code
Wants=network-online.target
After=network-online.target bluefin-dx-user-setup.service

[Service]
Type=oneshot
ExecStart=/usr/bin/bluefin-dx-vscode
Restart=on-failure
RestartSec=30
StartLimitInterval=0

[Install]
WantedBy=default.target
```